gradle/spec/fixtures/buildfiles/duck_duck_go_build.gradle.kts

```diff
@@ -0,0 +1,787 @@
+import org.jetbrains.kotlin.config.KotlinCompilerVersion
+
+apply(plugin = "com.android.application")
+
+plugins {
+    id("kotlin-android")
+    id("kotlin-android-extensions")
+    id("kotlin-kapt")
+}
+apply from = "../versioning.gradle"
+
+ext {
+    VERSION_NAME = "5.3.1"
+    USE_ORCHESTRATOR = project.hasProperty("orchestrator") ? project.property("orchestrator") : false
+}
+
+android {
+    compileSdkVersion(27)
+    defaultConfig {
+        applicationId = "com.duckduckgo.mobile.android"
+        minSdkVersion(21)
+        targetSdkVersion(27)
+        versionCode = buildVersionCode()
+        versionName = VERSION_NAME
+        testInstrumentationRunner = "com.duckduckgo.app.TestRunner"
+        archivesBaseName = "duckduckgo-$versionName" = versionName"
+
+        javaCompileOptions {
+            annotationProcessor(Options {)
+                arguments = listOf("room.schemaLocation" = "$projectDir/schemas".toString())
+            }
+        }
+        sourceSets {
+            androidTest.assets.srcDirs += files("$projectDir/schemas".toString())
+        }
+    }
+    signingConfigs {
+        release
+    }
+    buildTypes {
+        named("debug"){
+        }
+        named("release"){
+            isMinifyEnabled = false
+            setProguardFiles(listOf(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"))
+            signingConfig = signingConfigs.getByName("release")
+        }
+    }
+    externalNativeBuild {
+
+        cmake {
+            path "CMakeLists.txt"
+        }
+    }
+    lintOptions {
+        isAbortOnError = false
+    }
+    testOptions {
+        unitTests.returnDefaultValues = true
+
+        if (USE_ORCHESTRATOR) {
+            execution "ANDROID_TEST_ORCHESTRATOR"
+        }
+    }
+
+    apply(plugin = "com.android.application")
```

Comment:
Question: does the file start repeating here?

If that's an important feature to test, it may be worth a comment.

Reply:
Sorry, It is a bug in the auto converter that I used to convert from groovy to kotlin. I'll fix it manually.